src/style.rs

```diff
@@ -297,7 +359,24 @@ impl TabBarStyle {
 }
 
 impl TabStyle {
-    /// Derives relevant fields from `egui::Style` and sets the remaining fields to their default values.
+    ///Derives tab styles from `egui::Style`.
+    ///
+    /// See also: [`TabStyle::from_egui_active`], [`TabStyle::from_egui_inactive`],
+    /// [`TabStyle::from_egui_focused`], [`TabStyle::from_egui_hovered`], [`TabBodyStyle::from_egui`],
+    pub fn from_egui(style: &egui::Style) -> TabStyle {
+        Self {
+            active: TabInteractionStyle::from_egui_active(style),
+            inactive: TabInteractionStyle::from_egui_inactive(style),
+            focused: TabInteractionStyle::from_egui_focused(style),
+            hovered: TabInteractionStyle::from_egui_hovered(style),
+            tab_body: TabBodyStyle::from_egui(style),
+            ..Default::default()
+        }
+    }
+}
+
+impl TabInteractionStyle {
+    /// Derives relevant fields from `egui::Style` for an active tab and sets the remaining fields to their default values.
     ///
     /// Fields overwritten by [`egui::Style`] are:
```

Comment:
This list (as well as all those above the rest of this `impl`'s methods) is incorrect:
- It refers to fields of `TabStyle` that are no longer there
- It mentions fields that aren't being overwritten in this function.